Services/user/Api/Controllers/user.js

```diff
@@ -0,0 +1,50 @@
+const { User } = require('../../database/models');
+
+class UserController {
+  async createUser(call, callback) {
+    try {
+      const newUser = await User.create(call.request);
+      const payload = {
+        id: newUser.dataValues.id,
+        email: newUser.dataValues.email,
+        firstName: newUser.dataValues.firstName,
+        lastName: newUser.dataValues.lastName,
+      };
+      callback(null, payload);
+    } catch (error) {
+      callback(error);
+    }
+  }
+  async getUser(call, callback) {
+    try {
+      const getUser = await User.findById(call.request.id);
+      const payload = {
+        id: getUser.dataValues.id,
+        email: getUser.dataValues.email,
+        firstName: getUser.dataValues.firstName,
+        lastName: getUser.dataValues.lastName,
+      };
+      callback(null, payload);
+    } catch (error) {
+      callback(error);
+    }
+  }
+  async UpdateUser(call, callback) {
+    try {
+      await User.upsert(call.request);
```

Comment:
Change this to update or a find and update.

Reply:
I change it and I use update only